bin/clang/astVisitor.h

```diff
@@ -933,6 +933,13 @@ class SkeletonASTVisitor : public clang::RecursiveASTVisitor<SkeletonASTVisitor>
    */
   const clang::Decl* getOriginalDeclaration(clang::VarDecl* vd);
 
+  /**
+    @param path The path to a header to check if included in a systme folder
+    @param loc  The source location the include came from
+    @return Whether this is a "system" header that should be ignored in source-to-source
+  */
+  bool isSystemHeader(clang::SourceLocation loc, const std::string& path);
```

Comment:
Change name to `isInSystemHeader` mirrors https://clang.llvm.org/doxygen/classclang_1_1FullSourceLoc.html#a98ffaccf4e6822a7ffced0d7f6b4aee1